tests/integration.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Vue from 'vue';
+import { mount } from '@vue/test-utils';
+import CKEditor from '../src/index';
```

Comment:
I would rename it into `CKEditorComponent` like it has been done for `tests/component.js` file.

Reply:
Two different classes are exported.
- In `src/ckeditor.js` is `CKEditorComponent` - component
- In `src/index.js` is `CKEditor` - plugin which installs the component

In given case we are importing the plugin, so I'm keeping the `CKEditor` name to distinguish it from the component.